internal/cmd/mackupcmd_darwin.go

```diff
@@ -122,49 +122,42 @@ func (c *Config) runMackupAddCmd(
 }
 
 func (c *Config) mackupApplicationsDir() (chezmoi.AbsPath, error) {
-	brewPrefixCmd := exec.Command("brew", "--prefix")
-	brewPrefixData, err := brewPrefixCmd.Output()
+	mackupBinaryPath, err := exec.LookPath("mackup")
 	if err != nil {
 		return chezmoi.EmptyAbsPath, err
```

Comment:
I wonder if a better error than `chezmoi: exec: "mackup": executable file not found in $PATH` might be better than this.

`return chezmoi.EmptyAbsPath, fmt.Errorf("Mackup not installed (%s)", err)`?